src/components/UI/Navigation/MainMenu.js

```diff
@@ -0,0 +1,187 @@
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { CSSBreakpoints } from 'shared/constants';
+import {
+  AppCatalogRoutes,
+  AppRoutes,
+  OrganizationsRoutes,
+  UsersRoutes,
+} from 'shared/constants/routes';
+import { mq } from 'styles';
+import DropdownMenu from 'UI/DropdownMenu';
+
+import Hamburger from './Hamburger';
+
+const NavDiv = styled.div`
+  float: left;
+  padding-left: 20px;
+  height: 40px;
+
+  & > a {
+    text-decoration: none;
+    color: #eee;
+    margin-right: 18px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &:hover {
+      color: ${(props) => props.theme.colors.white1};
+    }
+  }
+
+  @media only screen and (max-width: ${(props) =>
```

Comment:
Why is this not the same as the component below?
```javascript
${mq(CSSBreakpoints.Large)} {
  display: flex;
}
```

Reply:
because 🤦‍♂!